examples/kotlin/src/main/kotlin/org/androidannotations/gradle/activity/HelloAndroidActivity.kt

```diff
@@ -43,3 +44,19 @@ open class HelloAndroidActivity : Activity() {
         helloTextView.text = helloMessage
     }
 }
+
+/**
+ * As of Kotlin 1.0.6+ and AndroidAnnotations 4.4.0, you can use the kotlin-allopen plugin
+ * to remove the need to explicitly declare enhanced classes or methods as <code>open</code>.
+ *
+ * See the build.gradle file or [the Kotlin Documentation](https://kotlinlang.org/docs/reference/compiler-plugins.html#all-open-compiler-plugin)
```

Comment:
I would rather link to our kotlin suppor twiki page. We are not having one, but we should open it. :)
(We should list the `KotlinOpen` annotation there for example.)

Also we could add some gotchas for Kotlin, like `lateinit` and `@JvmField`.